### PR TITLE
commit view page

### PR DESCRIPTION
**Pull request by TarasMel**
All done

### CODE REVIEW

File: app/src/main/java/com/example/taras/gascalculation/LauncherFragment.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+ * Activities that contain this fragment must implement the
+ * {@link LauncherFragment.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ * Use the {@link LauncherFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class LauncherFragment extends Fragment {
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
```
Comment (raa2911) [inline app/src/main/java/com/example/taras/gascalculation/LauncherFragment.java]: Remove default parameters and methods if not use it

File: app/src/main/java/com/example/taras/gascalculation/MainActivity.java [hunk 2]
```diff
@@ -1,19 +1,58 @@
...
-import android.widget.EditText;
-import android.widget.Toast;
+import android.support.design.widget.TabLayout;
+import android.support.v4.view.ViewPager;
+
+public class MainActivity extends BaseActivity implements
+        LauncherFragment.OnFragmentInteractionListener,
+        ResultFragment.OnFragmentInteractionListener{
+
+    TabLayout tabLayout;
```
Comment (raa2911) [inline app/src/main/java/com/example/taras/gascalculation/MainActivity.java]: can be private

File: app/src/main/java/com/example/taras/gascalculation/PagerAdapter.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentStatePagerAdapter;
+
+/**
+ * Created by Taras on 20.02.2018.
+ */
+
+public class PagerAdapter extends FragmentStatePagerAdapter{
+
+    int mNumOfTabs;
```
Comment (raa2911) [inline app/src/main/java/com/example/taras/gascalculation/PagerAdapter.java]: Need be private

File: app/build.gradle [hunk 4]
```diff
@@ -21,13 +21,15 @@ android {
...
+    compile fileTree(include: ['*.jar'], dir: 'libs')
     androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-//    compile 'com.android.support:multidex:1.0.1'
+    //    compile 'com.android.support:multidex:1.0.1'
     compile 'com.android.support:multidex:1.0.0'
     compile 'com.android.support:appcompat-v7:25.0.0'
     compile 'com.android.support.constraint:constraint-layout:1.0.2'
+    compile 'com.android.support:support-v4:25.3.1'
```
Comment (raa2911) [inline app/build.gradle]: 1. use same version of library for support group.

2. add variable with support library version, like 
https://segunfamisa.com/posts/android-gradle-extra-properties

**Review by raa2911 — COMMENTED** [review]
Please complete the fixes

**Comment by TarasMel** [comment]
All done

**Review by raa2911 — APPROVED** [review]

**Pull request merged**